File: MODUL 4/PRAK304-2210817210032-fransiskusassisiindrawijaya.c
#include<stdio.h>
int main(){
int n;
scanf("%d",&n);
if (n>=1 && n<=9)
{printf("Satuan");}
else if (n>=10 && n<=19)
{printf("Belasan");}
else if (n>=20 && n<=99)
{printf("Puluhan");}
else if (n<1)
{printf("nol");}
else
{printf("Anda Menginput Melebihi Limit Bilangan");}
return 0;}

File: MODUL 4/PRAK304-2210817210032-fransiskusassisiindrawijaya.py
n = int(input())
if n>=1 and n<=9:
   print("Satuan")
elif n>=10 and n<=19:
   print("Belasan")
elif n>=20 and n<=99:
   print("Puluhan")
elif n<1:
   print("nol")
else:
   print("Anda Menginput Melebihi Limit Bilangan")
